src/client/src/apps/SpotlightRoute/Spotlight.tsx

```diff
@@ -173,7 +207,17 @@ export const Spotlight: FC = () => {
         onChange={handleChange}
         onKeyDown={handleOnKeyDown}
       />
-      <Response>{contacting ? loader : suggestions}</Response>
+      <Response>
+        {contacting ? (
+          loader
+        ) : (
+          <>
+            {directoryAppSuggestions}
```

Comment:
am I right that user here selects which of the registered apps is handling the intent? if so, I somehow assumed that it should not require user's action. If app is registered to handle the intent, then it just handles it silently without user action

Reply:
Also, currently, there are two ways to open the blotter (using fdc3, and directly). The fdc3 way doesn't support filters yet.

Reply:
ok, got it, I will then assume that it is for illustration only

Reply:
That's right